info.yml

```diff
@@ -43,12 +54,12 @@ configuration:
 
     movie_width:
         type: int
-        default_value: 1024
+        default_value: 1920
```

Comment:
While this is a big departure for the app, I talked with Rob and we're free to refactor as much as we want for this app as nobody uses it anyway.